fragment/fragment/src/androidTest/java/androidx/fragment/app/strictmode/FragmentStrictModeTest.kt

```diff
@@ -47,9 +47,10 @@ public class FragmentStrictModeTest {
 
     @Test
     public fun penaltyDeath() {
-        FragmentStrictMode.setDefaultPolicy(FragmentStrictMode.Policy.Builder()
+        val policy = FragmentStrictMode.Policy.Builder()
             .penaltyDeath()
-            .build())
+            .build()
+        FragmentStrictMode.setDefaultPolicy(policy)
```

Comment:
What Lint actually wanted you to do was:
```
FragmentStrictMode.setDefaultPolicy(
    FragmentStrictMode.Policy.Builder()
        .penaltyDeath()
        .build()
)
```

Reply:
Sure, I just thought this would be more readable. You want me to change it?

Reply:
We actually have a `codestyles.xml` that enforces the style ian mentioned just to have some consistency, does that not automatically configure for you when starting studio via `./gradlew studio`?

Reply:
I used `./gradlew studio`, but could be that I just didn't auto-format before committing.